VSCode/armsnippets.json

```diff
@@ -758,6 +758,213 @@
     ],
     "description": "Cosmos DB Database Account"
 },
+"Cosmos DB SQL Database": {
+    "prefix": "arm-cosmos-sql-database",
+    "body": [
+      "      {",
+      "            \"type\": \"Microsoft.DocumentDB/databaseAccounts/apis/databases\",",
+      "            \"name\": \"[concat(${1:account-name}, '/sql/', ${2:database-name})]\",",
+      "            \"apiVersion\": \"2016-03-31\",",
+      "            \"dependsOn\": [ \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', ${1:account-name})]\" ],",
+      "            \"properties\":{",
+      "                \"resource\":{",
+      "                    \"id\": \"${2:database-name}\"",
+      "                },",
+      "                \"options\": { \"throughput\": \"${3:throughput}\" }",
```

Comment:
Placeholder defaults values should be valid values for the property they're assigned to (I'm guessing this should be a number rather than "throughput"?)